projector-server/build.gradle.kts

```diff
@@ -189,12 +189,12 @@ val downloadMonoFonts by tasks.creating<Task> {
   doLast {
     downloadFontsInZip(
       "mono",
-      "https://download.jetbrains.com/fonts/JetBrainsMono-1.0.3.zip",
+      "https://github.com/JetBrains/JetBrainsMono/releases/download/v2.242/JetBrainsMono-2.242.zip",
```

Comment:
Is there a reason switching to github link from official site one's?
Also, it's better to extract `2.242` into a variable so only that variable change will be needed to update in future

Reply:
I am used to using GitHub 
i will change it

Reply:
Well, let's hope the archive layout won't change in the future :) Because if it does, then extracting the version doesn't look like making the code much better.

However I see you've already extracted it so let's leave it extracted `¯\_(ツ)_/¯`